django-backend/fecfiler/transactions/fixtures/test_transaction_manager_transactions.json

```diff
@@ -0,0 +1,225 @@
+[
+    {
+        "model": "contacts.contact",
+        "pk": "1578e90c-5348-4afa-9db8-cbeddf9aa701",
```

Comment:
Are the "pk" fields necessary here, or will they auto-populate?  Not that it would make a significant difference either way